cmd/mcu_com.c

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
`Copyright` ?

Reply:
Taken from 9f0e582239916661ca3faa64ca9a7788f80990d9, we should fix it there @htho-lgs

Reply:
As I guess your PR will be merged firs, I kindly ask you to take care of it